packages/docusaurus-plugin-content-docs/src/theme/DocPage/index.js

```diff
@@ -12,6 +12,8 @@ import Layout from '@theme/Layout'; // eslint-disable-line
 
 import DocSidebar from '@theme/DocSidebar';
 
+import './styles.css';
```

Comment:
Let's be consistent and use CSS Modules here as well. Just leave out the class names that come from the dependency library.